src/app/home-page/home-logged-out/home.component.html

```diff
@@ -60,223 +60,237 @@ <h2>An app store for bioinformatics</h2>
       </div>
     </div>
   </div>
-
-  <div class="container">
-    <mat-tab-group mat-align-tabs="center" class="purple-tab-group">
-      <mat-tab>
-        <ng-template mat-tab-label>
-          <div fxLayoutGap="0.5rem">
-            <img src="../../../assets/images/home/icons-cloud.svg" alt="cloud icon" />
-            <span>Use In the Cloud</span>
-          </div>
-        </ng-template>
-        <div fxLayout="row" fxLayoutAlign="space-between center" fxLayoutGap="3rem" class="mt-3">
-          <div fxLayout="column" fxLayoutGap="3rem" fxFlex>
-            <h3><span class="title-strong">Use</span> in many analysis environments.</h3>
-            <span class="subtitle"
-              >Try out the
-              <a [routerLink]="'/workflows/github.com/dockstore/bcc2020-training/HelloWorld:master?tab=info'">Hello World Workflow</a> in
-              the cloud.</span
-            >
-            <ul class="ordered-list">
-              <li>
-                Visit the
-                <a [routerLink]="'/workflows/github.com/dockstore/bcc2020-training/HelloWorld:master?tab=info'">Hello World Workflow</a>; a
-                tutorial from our
-                <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/watch?v=shMr_Bd01Ko&t=3226s">BCC 2020 workshop</a
-                >.
-              </li>
-              <li>
-                Try out the workflow by running it on a variety of launch partners including
-                <a target="_blank" rel="noopener noreferrer" href="https://www.dnastack.com/">DNAstack</a>,
-                <a target="_blank" rel="noopener noreferrer" href="https://terra.bio">Terra</a>,
-                <a target="_blank" rel="noopener noreferrer" href="https://www.dnanexus.com/">DNAnexus</a>, and
-                <a target="_blank" rel="noopener noreferrer" href="https://www.sevenbridges.com">SevenBridges</a>.
-              </li>
-            </ul>
-          </div>
-          <img src="../../../assets/images/home/diagram-launch-with.png" alt="diagram" fxHide.lt-md fxFlex="45" class="contain-img" />
-        </div>
-      </mat-tab>
-      <mat-tab>
-        <ng-template mat-tab-label>
-          <div fxLayoutGap="0.5rem">
-            <img src="../../../assets/images/home/icons-navigation-about.svg" alt="cloud icon" />
-            <span>Use Locally</span>
+  <div *ngIf="(toolCategories$ | async).length + (workflowCategories$ | async).length > 0" fxLayout="column" class="categories">
```

Comment:
what happens without the fxlayout?

Reply:
Looks like leftover cruft, doesn't do anything, removed.